topology/probes/neutron.go

```diff
@@ -163,7 +164,12 @@ func (mapper *NeutronProbe) retrieveAttributes(portMd portMetadata) (*attributes
 
 	var IPs []string
 	for _, element := range port.FixedIPs {
-		IPs = append(IPs, element.IPAddress)
+		subnet, err := subnets.Get(mapper.client, element.SubnetID).Extract()
+		if err != nil {
+			return nil, err
+		}
+		neutron_ips := element.IPAddress + "/" + strings.Split(subnet.CIDR, "/")[1]
```

Comment:
thanks for submitting this. could you please just change neutron_ips to neutronIPs mainly because the golinter will raise a warning. Otherwise it is ok for me

Reply:
okay， it is my fault